errorutil_analyze_errors.json

```diff
@@ -702,24 +774,35 @@
         "suggested_remediation": "Restart Meshery Server\nPlease check if Meshery server is accessible to the Database"
       }
     ],
+    "ErrEncodingCode": [
+      {
+        "name": "ErrEncodingCode",
+        "code": "",
+        "severity": "Alert",
+        "long_description": "",
+        "short_description": "",
+        "probable_cause": "",
+        "suggested_remediation": ""
+      }
+    ],
     "ErrGrafanaBoardsCode": [
       {
         "name": "ErrGrafanaBoardsCode",
         "code": "",
         "severity": "Alert",
         "long_description": "",
-        "short_description": "",
+        "short_description": "unable to get grafana boards",
         "probable_cause": "",
-        "suggested_remediation": "Check if your Grafana endpoint is correct.\nConnect to Grafana from the settings page in the UI"
+        "suggested_remediation": "Check if your Grafana endpoint is correct\nConnect to Grafana from the settings page in the UI"
       },
       {
         "name": "ErrGrafanaBoardsCode",
         "code": "",
         "severity": "Alert",
         "long_description": "",
-        "short_description": "",
+        "short_description": "unable to get Prometheus boards",
         "probable_cause": "",
-        "suggested_remediation": "Check if your Prometheus endpoint is correct.\nConnect to Prometheus from the settings page in the UI"
+        "suggested_remediation": "Check if your Prometheus endpoint is correct\nConnect to Prometheus from the settings page in the UI"
```

Comment:
Why is punctuation removed? This is user-facing documentation.

Reply:
Hey @leecalcote so should I keep all the punctuation?